modules/simple_forms_api/lib/simple_forms_api.rb

```diff
@@ -39,6 +39,8 @@ def scrub_pii(message)
           words_to_remove += SimpleFormsApi::VBA210966.new(params).words_to_remove
         when '20-10207'
           words_to_remove += SimpleFormsApi::VBA2010207.new(params).words_to_remove
+        when '40-10007'
+          words_to_remove += SimpleFormsApi::VBA4010007.new(params).words_to_remove
```

Comment:
It looks like `SimpleFormsApi::VBA4010007` doesn't implement `words_to_remove` yet, so this won't have the desired effect until you implement that. You can look at one of the other forms that's implemented here, like this example: https://github.com/department-of-veterans-affairs/vets-api/blob/fad166bf6089dadbdef5b675bba159c7889d35af/modules/simple_forms_api/app/models/simple_forms_api/vba_21_0966.rb#L30-L34